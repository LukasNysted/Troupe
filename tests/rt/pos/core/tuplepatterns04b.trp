let fun f () = ((),(1),(2,3)) in
  case f () of
    ((),(),(x,3)) => x
  end