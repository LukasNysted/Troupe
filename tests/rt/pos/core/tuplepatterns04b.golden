2024-02-14T11:33:32.794Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread 254f9e5c-4e54-4722-b875-91761bf88ddb@{}%{}
>> pattern match failure in case expression at :2:3
